pages/dynamic-portal.tsx

```diff
@@ -591,7 +590,7 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   while (!(workbook || DateTime.now() > timeInFive)) {
     spaceData = await getSpace({
       workflow: WorkflowType.Dynamic,
-      spaceId: (existingSpace?.flatfileData as unknown as FlatfileSpaceData).id,
+      spaceId: (existingSpace?.flatfileData).id,
```

Comment:
Why don't we need this type casting anymore?

Reply:
We do need it. 

Moved interface to new flatfile file and reverted type casting removal